DSCResources/cChocoPackageInstall/cChocoPackageInstall.psm1

```diff
@@ -382,8 +401,10 @@ Function Upgrade-Package {
         [Parameter(Position=1)]
         [string]$pParams,
         [Parameter(Position=2)]
-        [string]$pSource,

```

Comment:
If you change the position of this parameter existing code, that don't explicitly use the parameter name, will break.